project/plugins.sbt

```diff
@@ -1,15 +1,7 @@
-credentials += Credentials(Path.userHome / ".sbt" / ".credentials")
+resolvers += Resolver.url("hmrc-sbt-plugin-releases",
+  url("https://dl.bintray.com/hmrc/sbt-plugin-releases"))(Resolver.ivyStylePatterns)
 
-val hmrcRepoHost = java.lang.System.getProperty("hmrc.repo.host", "https://nexus-preview.tax.service.gov.uk")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.9.0")
 
-resolvers ++= Seq(
-  "hmrc-snapshots" at hmrcRepoHost + "/content/repositories/hmrc-snapshots",
-  "hmrc-releases" at hmrcRepoHost + "/content/repositories/hmrc-releases",
-  Resolver.url("hmrc-sbt-plugin-releases",
-    url(hmrcRepoHost + "/content/repositories/bintray-hmrc-sbt-plugin-releases"))(Resolver.ivyStylePatterns)
-)
-
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "1.8.0")
-
-addSbtPlugin("io.gatling" % "gatling-sbt" % "2.2.0")
+addSbtPlugin("io.gatling" % "gatling-sbt" % "2.2.2")
```

Comment:
Latest version of the gatling sbt plugin is 3.1.0, any reason for not going to this? (The 3.x version of the sbt plugin seems to work fine with gatling 2.3.1)